src/ansible_navigator/actions/settings.py

```diff
@@ -0,0 +1,201 @@
+"""The ``settings`` subcommand action."""
+
+import curses
+
+from typing import Any
+from typing import Dict
+from typing import List
+from typing import Tuple
+
+from .._yaml import human_dump
+from ..action_base import ActionBase
+from ..action_defs import RunStdoutReturn
+from ..app_public import AppPublic
+from ..configuration_subsystem import transform_settings
+from ..steps import Step
+from ..ui_framework import CursesLinePart
+from ..ui_framework import CursesLines
+from ..ui_framework import Interaction
+from . import _actions as actions
+from . import run_action
+
+
+def filter_content_keys(obj: Dict[Any, Any]) -> Dict[Any, Any]:
+    """Filter out some keys when showing content.
+
+    :param obj: The object to be filtered
+    :return: The object with keys filtered out
+    """
+    return {k: v for k, v in obj.items() if not k.startswith("__")}
+
+
+def color_menu(colno: int, colname: str, entry: Dict[str, Any]) -> Tuple[int, int]:
+    # pylint: disable=unused-argument
+    """Color the menu.
+
+    :param colno: Column number
+    :param colname: Column name
+    :param entry: Column value
+    :return: Tuple int used to color the menu
+    """
+    if entry["is_default"]:
+        return 2, 0
+    return 3, 0
+
+
+def content_heading(obj: Any, screen_w: int) -> CursesLines:
+    """Create a heading for the setting entry showing.
+
+    :param obj: The content going to be shown
+    :param screen_w: The current screen width
+    :return: The heading
+    """
+    heading = []
+    string = obj["name"].replace("_", " ")
+    if obj["is_default"]:
+        string += f" (current/default: {obj['current_value']})"
+        color = 2
+    else:
+        string += f" (current: {obj['current_value']})  (default: {obj['default']})"
+        color = 3
+
+    string = string + (" " * (screen_w - len(string) + 1))
+
+    heading.append(
+        tuple(
+            [
+                CursesLinePart(
+                    column=0,
+                    string=string,
+                    color=color,
+                    decoration=curses.A_UNDERLINE,
+                ),
+            ],
+        ),
+    )
+    return tuple(heading)
```

Comment:
Looking closer, it looks like `heading = []` is created at the start of the function body only to be populated with a single item and then converted to a tuple. It seems to be overkill in this case. Why not just return this data structure with this single nested tuple like this?
```suggestion
    return (
        (
            CursesLinePart(
                column=0,
                string=string,
                color=color,
                decoration=curses.A_UNDERLINE,
            ),
        ),
    )
```

Reply:
This is most likely a carry over from other actions that have a multi line heading that is incrementally created but ultimately turned into a tuple.

This is cleaner, although my preference is that it be assigned to a variable and then that variable returned.  I find it easier to put in breakpoints in, debug and multi line return statements atypical in navigator.

Reply:
Well, this still means that `.append()` is overkill. Could be `heading = [that_huge_thing]` w/o having to maintain a detached variable definition when it's only used once.

Reply:
This has been resolved:
`    heading = (
        (
            CursesLinePart(
                column=0,
                string=string,
                color=color,
                decoration=curses.A_UNDERLINE,
            ),
        ),
    )

    return heading`